nextroute/schema/output.go

```diff
@@ -6,60 +6,96 @@ import (
 	"github.com/nextmv-io/sdk/nextroute/check"
 )
 
-// SolutionOutput represents a solutions as JSON.
+// SolutionOutput represents a solution as JSON.
 type SolutionOutput struct {
-	Unplanned []StopOutput    `json:"unplanned"`
-	Vehicles  []VehicleOutput `json:"vehicles"`
+	// Unplanned is the list of stops that were not planned in the solution.
+	Unplanned []StopOutput `json:"unplanned"`
+	// Vehicles is the list of vehicles that were used in the solution.
+	Vehicles []VehicleOutput `json:"vehicles"`
+	// Objective is the objective of the solution.
 	Objective ObjectiveOutput `json:"objective"`
-	Check     *check.Output   `json:"check,omitempty"`
+	// Check is the check of the solution.
+	Check *check.Output `json:"check,omitempty"`
 }
 
 // StopOutput is the basic struct for a stop.
 type StopOutput struct {
-	ID         string   `json:"id"`
-	Location   Location `json:"location"`
-	CustomData any      `json:"custom_data,omitempty"`
+	// ID is the ID of the stop.
+	ID string `json:"id"`
+	// Location is the location of the stop.
+	Location Location `json:"location"`
+	// CustomData is the custom data of the stop.
```

Comment:
I would add: "This will contain any custom data set by the user on the input stop." (or similar)